canonical-web-and-design/ubuntu.com
# Sharable cart referral link - get the data

## Done

- get and display the user's offers on the /offers page

## QA

- log in on https://ubuntu-com-11106.demos.haus/advantage?test_backend=true
- go to https://ubuntu-com-11106.demos.haus/advantage/offers?test_backend=true (with some offers on your account)
- check that they display correctly

## Issue / Card

Fixes #

## Screenshots

![image](https://user-images.githubusercontent.com/11927929/148189881-bc2a717b-6e4b-4d75-a214-52b30351c858.png)



## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
